src/Fields/Fields.jl

```diff
@@ -582,4 +580,52 @@ if VERSION < v"1.10"
     end
 end
 
+
+"""
+    field2array(field)
+
+Extracts a view of a `ClimaCore` `Field`'s underlying array. Can be used to
+simplify the process of getting and setting values in an `RRTMGPModel`; e.g.
+```
+    model.center_temperature .= field2array(center_temperature_field)
+    field2array(face_flux_field) .= model.face_flux
+```
+
+The dimensions of the resulting array are `([number of vertical nodes], number
+of horizontal nodes)`. Also, `field` must be a `Field` of scalars, so that the
+element type of the array is the same as the struct type of `field`.
+"""
+array2field(array, space) = Field(
+    DataLayouts.array2data(array, Spaces.local_geometry_data(space)),
+    space,
+)
```

Comment:
This docstring is mismatched with the actual function

Reply:
Ah, yeah, bummer, I’ll fix it